packages/jest/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import type { Transformer } from '@jest/transform';
+import { transformSync, getPackageInfo } from '@vanilla-extract/integration';
+import * as esbuild from 'esbuild';
+
+const vanillaTransformer: Transformer = {
+  canInstrument: false,
+  process(source, filePath, options) {
+    const { name: packageName } = getPackageInfo(options.config.rootDir);
+
+    const code = transformSync({
```

Comment:
The Jest transform has to be sync?

Reply:
Yeah apparently. Thought it's not that bad as no IO is actually happening here. 

https://github.com/facebook/jest/issues/11458#issuecomment-850454026